docs/.vuepress/config.js

```diff
@@ -154,6 +154,11 @@ module.exports = {
                         "/cloud/use/",
                         "/cloud/use/tailscale.md"
                     ]
+                },
+                {
+                    title:      "Faq",
+                    collapsable: false,
+                    path:        "/cloud/faq/",
```

Comment:
**FIX:** We either have to update the styling or add a nested child page. See:
![obraz](https://user-images.githubusercontent.com/5562528/108685092-0171a780-74f4-11eb-850e-db6602e43d75.png)